### PR TITLE
add blog k8s on mips

### PR DESCRIPTION
**Pull request by timyinshi**
add Chinese version of blog 《kubernetes on MIPS》

### CODE REVIEW

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
+---
+layout: blog
+title: "Kubernetes on MIPS"
```
[the remark this answers is not in the document]
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: ok,i have resolved it.

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+<!-- 
+## Background
+-->
+## 背景
+
+<!-- 
+[MIPS](https://en.wikipedia.org/wiki/MIPS_architecture) (Microprocessor without Interlocked Pipelined Stages) is a reduced instruction set computer (RISC) instruction set architecture (ISA), appeared in 1981 and developed by MIPS Technologies. Now MIPS architecture is widely used in many electronic products.
+-->
+[MIPS](https://en.wikipedia.org/wiki/MIPS_architecture) (Microprocessor without Interlocked Pipelined Stages) 是一种采取精简指令集（RISC）的处理器架构 (ISA),出现于 1981 年，由 MIPS 科技公司开发。如今 MIPS 架构被广泛应用于许多电子产品上。
```
[the remark this answers is not in the document]
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: ok,i have fixed it.

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
+<!--
+layout: blog
+title: "Kubernetes on MIPS"
+date: 2020-01-15
+slug: Kubernetes-on-MIPS
+-->
+---
+layout: blog
+title: "支持 MIPS 架构 Kubernetes 的方案"
```
Comment (chenrui333) [inline content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]:
```suggestion
title: "基于 MIPS 架构的 Kubernetes 方案"
```

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+
+<!-- 
+## Background
+-->
+## 背景
+
+<!-- 
+[MIPS](https://en.wikipedia.org/wiki/MIPS_architecture) (Microprocessor without Interlocked Pipelined Stages) is a reduced instruction set computer (RISC) instruction set architecture (ISA), appeared in 1981 and developed by MIPS Technologies. Now MIPS architecture is widely used in many electronic products.
+-->
+[MIPS](https://zh.wikipedia.org/wiki/MIPS%E6%9E%B6%E6%A7%8B) (Microprocessor without Interlocked Pipelined Stages) 是一种采取精简指令集（RISC）的处理器架构 (ISA),出现于 1981 年，由 MIPS 科技公司开发。如今 MIPS 架构被广泛应用于许多电子产品上。
```
Comment (chenrui333) [inline content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]:
```suggestion
[MIPS](https://zh.wikipedia.org/wiki/MIPS%E6%9E%B6%E6%A7%8B) (Microprocessor without Interlocked Pipelined Stages) 是一种采取精简指令集（RISC）的处理器架构 (ISA)，出现于 1981 年，由 MIPS 科技公司开发。如今 MIPS 架构被广泛应用于许多电子产品上。
```

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+
+<!--
+For many years, to enrich the ecology of the open-source community, we have been working on adjusting MIPS architecture for Kubernetes use cases. With the continuous iterative optimization and the performance improvement of the MIPS CPU, we have made some breakthrough progresses on the mips64el platform.
+-->
+多年来，为了丰富开源社区的生态，我们一直致力于在 MIPS 架构下适配 Kubernetes。随着 MIPS CPU 的不断迭代优化和性能的提升，我们在 mips64el 平台上取得了一些突破性的进展。
+
+<!-- 
+Over the years, we have been actively participating in the Kubernetes community and have rich experience in the using and optimization of Kubernetes technology. Recently, we tried to adapt the MIPS architecture platform for Kubernetes and achieved a new a stage on that journey. The team has completed migration and adaptation of Kubernetes and related components, built not only a stable and highly available MIPS cluster but also completed the conformance test for Kubernetes v1.16.2.
+-->
+多年来，我们一直积极投入 Kubernetes 社区，在 Kubernetes 技术应用和优化方面具备了丰富的经验。最近，我们在研发过程中尝试将 Kubernetes 适配到 MIPS 架构平台，并取得了阶段性成果。成功完成了 Kubernetes 以及相关组件的迁移适配，不仅搭建出稳定高可用的 MIPS 集群，同时完成了 Kubernetes-v1.16.2 的一致性测试。
```
Comment (chenrui333) [inline content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]:
```suggestion
多年来，我们一直积极投入 Kubernetes 社区，在 Kubernetes 技术应用和优化方面具备了丰富的经验。最近，我们在研发过程中尝试将 Kubernetes 适配到 MIPS 架构平台，并取得了阶段性成果。成功完成了 Kubernetes 以及相关组件的迁移适配，不仅搭建出稳定高可用的 MIPS 集群，同时完成了 Kubernetes v1.16.2 版本的一致性测试。
```

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+
+<!--
+## K8S-MIPS component build
+-->
+## K8S-MIPS 组件构建
+
+<!--
+Almost all native cloud components related to Kubernetes do not provide a MIPS version installation package or image. The prerequisite of deploying Kubernetes on the MIPS platform is to compile and build all required components on the mips64el platform. These components include:
+-->
+基本上所有的 Kubernetes 相关的云原生组件都没有提供 MIPS 版本的安装包或镜像，在 MIPS 平台上部署 Kubernetes 的前提是自行编译构建出全部所需组件。这些组件主要包括：
```
Comment (chenrui333) [inline content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]:
```suggestion
几乎所有的 Kubernetes 相关的云原生组件都没有提供 MIPS 版本的安装包或镜像，在 MIPS 平台上部署 Kubernetes 的前提是自行编译构建出全部所需组件。这些组件主要包括：
```

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+- `gcr.io/kubernetes-e2e-test-images/volume/iscsi:2.0`
+- `gcr.io/kubernetes-e2e-test-images/volume/nfs:1.0`
+- `gcr.io/kubernetes-e2e-test-images/volume/rbd:1.0.1`
+- `k8s.gcr.io/etcd:3.3.15`
+- `k8s.gcr.io/pause:3.1`
+
+<!--
+Finally, we ran the tests and got the test result, include `e2e.log`, which showed that all test cases passed. Additionally, we submitted our test result to [k8s-conformance](https://github.com/cncf/k8s-conformance) as a [pull request](https://github.com/cncf/k8s-conformance/pull/779).
+-->
+最终我们执行一致性测试并且得到了测试报告，包括 `e2e.log`，显示我们通过了全部的测试用例。此外，我们将测试结果以 [pull request](https://github.com/cncf/k8s-conformance/pull/779) 的形式提交到了 [k8s-conformance](https://github.com/cncf/k8s-conformance) as a 。
```
Comment (chenrui333) [inline content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]:
```suggestion
最终我们执行一致性测试并且得到了测试报告，包括 `e2e.log`，显示我们通过了全部的测试用例。此外，我们将测试结果以 [pull request](https://github.com/cncf/k8s-conformance/pull/779) 的形式提交给了 [k8s-conformance](https://github.com/cncf/k8s-conformance)。
```

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+
+<!--
+## What's next
+-->
+## 后续计划
+
+<!--
+We built the kubernetes-MIPS component manually and finished the conformance test, which verified the feasibility of Kubernetes On the MIPS platform and greatly enhanced our confidence in promoting the support of the MIPS architecture by Kubernetes.
+-->
+我们手动构建了 K8S-MIPS 组件以及执行了 E2E 测试，验证了 Kubernetes On MIPS 的可行性，极大的增强了我们对于推进 Kubernetes 支持 MIPS 架构的信心。
```
Comment (chenrui333) [inline content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]:
```suggestion
我们手动构建了 K8S-MIPS 组件以及执行了 E2E 测试，验证了 Kubernetes on MIPS 的可行性，极大的增强了我们对于推进 Kubernetes 支持 MIPS 架构的信心。
```

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+
+<!--
+In the future, we plan to actively contribute our experience and achievements to the community, submit PR, and patch for MIPS. We hope that more developers and companies in the community join us and promote Kubernetes on MIPS.
+-->
+后续，我们将积极地向社区贡献我们的工作经验以及成果，提交 PR 以及 Patch For MIPS 等， 希望能够有更多的来自社区的力量加入进来，共同推进 Kubernetes For MIPS 的进程。
```
Comment (chenrui333) [inline content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]:
```suggestion
后续，我们将积极地向社区贡献我们的工作经验以及成果，提交 PR 以及 Patch For MIPS 等， 希望能够有更多的来自社区的力量加入进来，共同推进 Kubernetes for MIPS 的进程。
```

**Comment by chenrui333** [comment]
Also, please update the commit message to better reflect the chinese translation part :), maybe like `zh-trans: add k8s on mips blog post`

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md (hunk 3 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: fixed it

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md (hunk 4 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: fixed  it

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md (hunk 5 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: fixed it

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md (hunk 7 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: fixed it

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md (hunk 8 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: fixed it

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md (hunk 9 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: fixed it

File: content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md (hunk 6 above)
Comment (timyinshi) [reply content/zh/blog/_posts/2020-01-15-Kubernetes-on-MIPS.md]: fixed it

**Pull request merged**